src/ui/page/settings/view.jsx

```diff
@@ -188,7 +201,41 @@ class SettingsPage extends React.PureComponent<Props, State> {
             </section>
 
             <section className="card card--section">
-              <h2 className="card__title">{__('Max Purchase Price')}</h2>
+              <h2 className="card__title">{__('Network and Data Settings')}</h2>
+
+              <Form>
+                <FormField
+                  type="checkbox"
+                  name="save_files"
+                  onChange={() => setDaemonSetting('save_files', !daemonSettings.save_files)}
+                  checked={daemonSettings.save_files}
+                  label={__(
+                    'Enables saving of all viewed content to your downloads directory. Paid content and some file types are saved by default.'
```

Comment:
Check boxes always enable, so you do not need to put this in the label. Present tense is better:

"Save all viewed content"

Helper text:

"Content is saved to downloads directory. Paid content and some file types are saved by default. Changing this setting will not affect previously downloaded content."

Possibly the first sentence of helper would be better in label, or something close to this. Try both.

Reply:
Nice. Much cleaner